src/routes/analysis/insights/+page.svelte

```diff
@@ -89,7 +89,7 @@
     <button class="rounded bg-gunmetal p-2" onclick={() => goto("/home")}
         ><Home /></button
     >
-    <span class="self-center align-middle">Analysis Page</span>
+    <span class="flex flex-1 justify-center self-center">Analysis Page</span>
```

Comment:
```suggestion
    <span class="flex-1 text-center">Analysis Page</span>
```
self center is for grid childern